ngi_pipeline/database/filesystem.py

```diff
@@ -160,9 +166,18 @@ def create_charon_entries_from_project(project, best_practice_analysis="whole_ge
                                      'seqrun "{}" already exists; next...'.format(project, sample,
                                                                                   libprep, seqrun))
                     else:
+                        update_failed=True
                         LOG.error(e)
                         continue
 
+    if update_failed :
+        if retry_on_fail:
+        create_charon_entries_from_project(project, best_practice_analysis=best_practice_analysis,
```

Comment:
it's only github showing the line as a break, if you look, there's no line number.